packages/apputils/src/sanitizer.ts

```diff
@@ -569,7 +569,10 @@ class Sanitizer implements ISanitizer {
       'ul',
       'var',
       'video',
-      'wbr'
+      'wbr',
+      'svg',
+      'line',
+      'path'
```

Comment:
it would be nice if we offered enough capacity to at least display all of the svgs from our icon zoo this way. For that, we'll also need at least
- `g`
- `circle`
    - attrs: `[cx, cy, r, pathLength]`
- `polygon`
    - attrs: `[points, pathLength]`
- `rect`
    - attrs: `[x, y, width, height, rx, ry, pathLength]`

All of the above (including `g`) should also get these attrs:

```
[fill, fill-opacity, stroke, stroke-opacity, stroke-width]
```

Reply:
oh, also, you should alphabetize these tags, like the rest of the `allowedTags` array